pkg/minikube/sysinit/systemd.go

```diff
@@ -123,7 +125,14 @@ func (s *Systemd) Stop(svc string) error {
 
 // ForceStop terminates a service with prejudice
 func (s *Systemd) ForceStop(svc string) error {
-	_, err := s.r.RunCmd(exec.Command("sudo", "systemctl", "stop", "-f", svc))
+	rr, err := s.r.RunCmd(exec.Command("sudo", "systemctl", "stop", "-f", svc))
+	if err == nil {
```

Comment:
this if statement isn't really necessary, `return err` will return nil if err is nil

Reply:
agree with @sharifelgamal  this "if" is not needed !

Reply:
This `if` lets us to avoid matching substrings in `strings.Contains`. Also the whole logic is clearer with this shortcut - if `systemctl` returned ok - return immediately as no extra processing required

Reply:
this "if statement" adds to the code complexity in terms of number of If statments. and makes the code harder to read.

this is not inline with go style used in minikube, I suggest getting rid of it

Reply:
just curious, how it makes the code harder to read?

Reply:
because following the code we dont expect to check for err nil and return while it already returns,
a typical go reader with following the styles is used to look at the return and it is obvisous if it is nil it returns nil too,

plus it adds to the go "gocyclo" score https://goreportcard.com/report/github.com/kubernetes/minikube
so the less If statments, the better

Reply:
`because following the code we dont expect to check for err nil and return while it already returns,
a typical go reader with following the styles is used to look at the return and it is obvisous if it is nil it returns nil too.`  I don't understand this sentence

Cyclomatic complexity metric evaluates the number of branches inside the function, it's goal is to prevent functions overgrow. This method is 7 lines long, talking about cyclomatic complexity makes no sense here.

Reply:
this is not inline with minikube code style, it doesn't make sense to check for "nil" and return while we are returning in next line, this an example of code seems easier to write but harder to maintain 

I suggest reading more minikube code or  https://www.kubernetes.dev/docs/guide/coding-convention/

Reply:
I don't personally think this is more complex, it's just an awkward line of code for a small optimization.